src/2025h1/README.md

```diff
@@ -55,11 +55,13 @@ The flagship goals proposed for this roadmap are as follows:
 
-[**Rust for Linux.**](./rfl.md) The [experimental support for Rust development in the Linux kernel][https://rust-for-linux.com/] is a watershed moment for Rust, demonstrating to the world that Rust is indeed a true alternative to C. Currently the Linux kernel support depends on a wide variety of unstable features in Rust; these same features block other embedded and low-level systems applications. We are working to stabilize all of these features so that RFL can be built on a stable toolchain. As we have successfully stabilized the majority of the language features used by RFL, we plan in 2025H1 to turn our focus to compiler flags and tooling options. We will (a) implement RFC #3716 which lays out a design for ABI-modifying flags; (b) take the first step towards stabilizing [`build-std`](https://doc.rust-lang.org/cargo/reference/unstable.html#build-std) by [creating a stable way to rebuild core with specific compiler options](./build-std.md); (c) extending rustdoc, clippy, and the compiler with features that extract metadata for integration into other build systems (in this case, the kernel's build system).
+[**Rust for Linux.**](./rfl.md) The [experimental support for Rust development in the Linux kernel][https://rust-for-linux.com/] is a watershed moment for Rust, demonstrating to the world that Rust is indeed a true alternative to C. Currently the Linux kernel support depends on a wide variety of unstable features in Rust; these same features block other embedded and low-level systems applications. We are working to stabilize all of these features so that RFL can be built on a stable toolchain. As we have successfully stabilized the majority of the language features used by RFL, we plan in 2025H1 to turn our focus to compiler flags and tooling options. We will (a) implement [RFC 3716] which lays out a design for ABI-modifying flags; (b) take the first step towards stabilizing [`build-std`](https://doc.rust-lang.org/cargo/reference/unstable.html#build-std) by [creating a stable way to rebuild core with specific compiler options](./build-std.md); (c) extending rustdoc, clippy, and the compiler with features that extract metadata for integration into other build systems (in this case, the kernel's build system).
```

Comment:
Tangential: is `[experimental support for Rust development in the Linux kernel][https://rust-for-linux.com/]` working? Or should those be parenthesis? (it is not working at least in the GitHub render here).

Reply:
it doesn't https://rust-lang.github.io/rust-project-goals/2025h1/index.html#why-these-particular-flagship-goals so I'll fix it

Reply:
it should be fixed now

Reply:
Thanks!